tests/unit/stores/appStore.spec.ts

```diff
@@ -227,6 +226,60 @@ describe("app store", () => {
       expect(store.downloading).toBe(false);
     });
 
+    it("can set the current comparison based on the axis, baseline parameter, and selected scenario options", async () => {
+      const store = useAppStore();
+      store.setComparison("vaccine", { country: "USA", hospital_capacity: "54321", vaccine: "high", response: "elimination" }, ["none", "low"]);
+
+      expect(store.currentComparison).toEqual({
```

Comment:
Could make this less verbose by comparing each scenario with empty scenario spread with the expected parameter values.